src/realmd/AuthSocket.cpp

```diff
@@ -117,6 +117,7 @@ struct sAuthLogonProof_C
     uint8   crc_hash[20];
     uint8   number_of_keys;
     uint8   securityFlags;                                  // 0x00-0x04
+    PINData pinData;
```

Comment:
Dont you think that if you are doing memory magic there should be a code comment at least mentioning it?

Reply:
yeah, fair enough, I'll add a comment